pallets/runtime/mainnet/src/runtime.rs

```diff
@@ -332,6 +335,7 @@ construct_runtime!(
         CorporateBallot: pallet_corporate_ballot::{Module, Call, Storage, Event<T>} = 47,
         CapitalDistribution: pallet_capital_distribution::{Module, Call, Storage, Event<T>} = 48,
         Checkpoint: pallet_checkpoint::{Module, Call, Storage, Event<T>, Config} = 49,
+        Base: pallet_base::{Module, Call, Event} = 50,
```

Comment:
Do we need `Call` here? I think not

Reply:
Not sure, but either way, I'd prefer to reserve that index for base extrinsics we might want in the future.